CHANGELOG.md

```diff
@@ -96,7 +97,9 @@ Adding a new version? You'll need three changes:
  - [0.0.5](#005)
  - [0.0.4 and prior](#004-and-prior)
 
-## Unreleased
+## [3.3.0]
+
+> Release date: 2024-08-26
```

Comment:
Should it be 08-27?

Reply:
The code was checked out on 08.26 and has not been modified. The current lack of merge is only due to some issues with the CI process. Let's keep it as it is on 08.26 and stick to our original plan.

In addition, if considering the time zone, there are still some places at 08.26. :rofl:

Reply:
In Niue it is still 08.26 :-)